akka-docs-dev/rst/scala/stream-flows-and-basics.rst

```diff
@@ -161,7 +161,7 @@ Fast Publisher, slow Subscriber
 This is the case when back-pressuring the ``Publisher`` is required, because the ``Subscriber`` is not able to cope with
 the rate at which its upstream would like to emit data elements.
 
-Since the ``Publisher`` is not allowed to signal more elements than the pending demand signalled by the ``Subscriber``,
+Since the ``Publisher`` is not allowed to produce more elements than the pending demand signalled by the ``Subscriber``,
```

Comment:
Takes a bit of getting used to I guess, here are the words that can be used instead of signal: https://github.com/reactive-streams/reactive-streams#notes